test_vm/Cheffile

```diff
@@ -8,3 +8,4 @@ cookbook 'dbfit_test', :path => "vendor/cookbooks/dbfit_test"
 cookbook 'database', :git => "https://github.com/dbfit/database"
```

Comment:
Ok, everything updated and tested apart from this now. I'll have to do that tomorrow.